diff --git a/image.c b/image.c
index 03c89a9..0f641e5 100644
--- a/image.c
+++ b/image.c
@@ -255,7 +255,7 @@ drawImage(void)
 	    } else if (enable_inline_image == INLINE_IMG_ITERM2) {
 		put_image_iterm2(url, x, y, sw, sh);
 	    } else if (enable_inline_image == INLINE_IMG_KITTY) {
-		put_image_kitty(url, x, y, i->width, i->height, i->sx, i->sy, sw * pixel_per_char, sh * pixel_per_line_i, sw, sh);
+		put_image_kitty(url, x, y, i->cache->width, i->cache->height, i->sx, i->sy, sw * pixel_per_char, sh * pixel_per_line_i, sw, sh);
 	    }
 
 	    continue ;
